app/services/__tests__/LocationService.spec.js

```diff
@@ -1,5 +1,8 @@
+import BackgroundGeolocation from '@mauron85/react-native-background-geolocation';
+
 import * as General from '../../helpers/General';
-import { LocationData } from '../LocationService';
+import { LocationData, Reason } from '../LocationService';
+import LocationServices from '../LocationService';
 
 function mockGetStoreData(data) {
   General.GetStoreData = () => {
```

Comment:
To that end, I highly recommend adding a few methods to a  beforeEach hook after your describe block(s), to clear the state of the mocks between tests. This will ensure that the tests are not dependent on each other's data.
```
beforeEach(() => {
    jest.resetAllMocks();
    jest.resetAllModules();
});
```